docs/styles/README.md

```diff
@@ -3,28 +3,28 @@
 Styles passed into the Modal via the `style` prop are merged with the defaults.  The default styles are:
 
 ```js
```

Comment:
I think this is the last thing to get this merged. 

You can use change the sintax to `jsx` and add the component `<Modal ... />`.

Reply:
ah ok, yep